react-workshop/src/components/news-list.js

```diff
@@ -0,0 +1,58 @@
+import React, {Component} from 'react';
+import '../App.css';
+
+import NewsListItem from './news-list-item';
+
+const initalList = [
+    {
+    title: "react",
+    url: 'https://github.com/Nickybutt/ReactApp/pull/',
+    author: 'Jordan Walke',
+    num_comments: 3,
+    points: 4,
+    objectID: 0,
+  }, {
+    title: 'Redux',
+    url: 'https://github.com/reactjs/redux',
+    author: 'Dan Abramov, Andrew Clark',
+    num_comments: 2,
+    points: 5,
+    objectID: 1,
+  }, ];
+
+class NewsList extends Component {
+constructor(props) {
+super(props)
+
+this.state = {list:[...initalList]}
+};
+
+removeItem = objectID => () => {
```

Comment:
dit is een rare arrow functie, het verbaast mij dat het werkt. de functie zou moeten zijn

`(objectId) => { .... }`